packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -721,6 +729,8 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       return this.submitToCallDiagnosticsPreLogin(diagnosticEvent, options?.preLoginId);
     }
 
+    this.validator({type: 'ce', event: diagnosticEvent});
```

Comment:
Validator is a promise, this will simply be skipped and the resolve of `this.validator()` is ignored by the stack.

Reply:
on purpose

Reply:
So, it's a non-blocking async call to emit logging from the consumer side? I get this, but shouldn't we just halt the stack and **not** send the payload if its invalid?

I do understand that we can simply let it get blocked by CA if invalid, but just want to confirm.

Reply:
yes, we dont want to halt the stack, just log a warning because the events submitted to CA will still proceed fine, but should be fixed. This is a developer tool, not a blocking call.